### ISSUE TITLE
Possible typos

### ISSUE THREAD

**Issue by antecedent** [unimorph/ces#1]
Upon doing some inflectional class clustering, I noticed some rather odd outliers, which I, with my limited knowledge of Czech, assume to be typos inherited from Wiktionary.

Of course, native speakers or experts should have their say before actual changes are made.

The oddities follow:

- _bezpočnosti_ as loc. pl. of _bezpečnost_ (→ bezpečnosti?)
- _bohozlužba_ as nom. sg. of _bohoslužba_ (z → s, almost certainly, as nom. sg. _is_ the lemma)
- _krudiáty_ as nom. pl. of _kruciáta_ (d → c?)
- _puzlům_ as dat. pl. of _puzzle_ (z → zz?)
- _hozhovoru_ as loc. sg. of _rozhovor_ (→ rozhovoru?)
- _rpzumy_ as nom. pl. of _rozum_ (p → o, almost certainly)
- _zvazcích_ as loc. pl. of _svazek_ (→ svazcích?)

**Comment by dan-zeman** [comment]
> Of course, native speakers or experts should have their say before actual changes are made.

Yes, these are all errors, and the correct forms are as you suggest.
